### PR TITLE
900ZD template

### PR DESCRIPTION
**Pull request by bpholden**
Added new 900ZD with a recipe on how to make it, an update to keck_deimos to use it, and an update to build_template to fix a minor bug

### CODE REVIEW

File: doc/spectrographs.rst [hunk 1]
```diff
@@ -11,49 +11,7 @@ Below we describe all of the spectrographs that may
...
-vlt_fors2                 VLT         FORS2 spectrometer; only a few gratings
-vlt_xshooter_uvb          VLT         :doc:`xshooter` spectrometer; UVB camera
-vlt_xshooter_vis          VLT         :doc:`xshooter` spectrometer; VIS camera
-vlt_xshooter_nir          VLT         :doc:`xshooter` spectrometer; NIR camera
-wht_isis_blue             WHT         ISIS spectrometer; blue camera?
-p200_dbsp_blue            P200        DBSP spectrograph; blue camera
-p200_dbsp_red             P200        DBSP spectrograph; red camera
-p200_tspec                P200        TripleSpec spectrograph
-======================    =========   =======================================
+======================  =========   =======================================
```
Comment (profxj) [inline doc/spectrographs.rst:14]: We don't want to delete this list.

File: paper/astropy.txt [hunk 2]
```diff
@@ -1,28 +0,0 @@
-Title: PypeIt: The Python Spectroscopic Data Reduction Pipeline
-
-Authors: J. Xavier Prochaska (1, 2), Joseph F. Hennawi (3), Kyle B. Westfall
```
Comment (profxj) [inline paper/astropy.txt:3]: This file should not have been deleted.

Comment (profxj) [reply paper/astropy.txt:3]: Nor the figures that went with the paper.

File: pypeit/coadd1d.py [hunk 3]
```diff
@@ -160,7 +160,7 @@ def __init__(self, spec1dfiles, objids, spectrograph=None, par=None, sensfile=No
             header = fits.getheader(spec1dfiles[0])
             self.spectrograph = load_spectrograph(header['PYP_SPEC'])
         if par is None:
-            self.par = self.spectrograph.default_pypeit_par()['coadd1d']
+            self.par = spectrograph.default_pypeit_par()['coadd1d']
```
Comment (profxj) [inline pypeit/coadd1d.py:163]: Did you touch this line?

File: pypeit/core/framematch.py [hunk 4]
```diff
@@ -106,7 +106,7 @@ def check_frame_exptime(exptime, exprng):
     if exprng[0] is not None:
         indx[indx] &= (exptime[indx] > exprng[0])
     if exprng[1] is not None:
-        indx[indx] &= (exptime[indx] <= exprng[1])
+        indx[indx] &= (exptime[indx] < exprng[1])
```
Comment (profxj) [inline pypeit/core/framematch.py:109]: Or this?

File: pypeit/data/extinction/palomarextinct.dat [hunk 5]
```diff
@@ -1,36 +0,0 @@
-# taken from http://articles.adsabs.harvard.edu/pdf/1975ApJ...197..593H
```
Comment (profxj) [inline pypeit/data/extinction/palomarextinct.dat:1]: Was this meant to be deleted?

**Review by profxj — CHANGES_REQUESTED** [review]
A bunch of things got munged in this PR...

We probably need to start over..

**Comment by bpholden** [comment]
borked.

**Pull request closed without merging**